app/views/password_resets/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Reset your password</h1>
+<%= form_for @user, url: password_reset_path(params[:id]) do |f| %>
+  <% if @user.errors.any? %>
+    <div class="error_msg">
+      <ul>
+        <% for msg in @user.errors.full_messages %>
```

Comment:
In Ruby, favor using `.each` for iteration instead of `for ... in ...` as its a little more conventional.